networking/aws_load_balancer_operator/installing-albo-sts-cluster.adoc

```diff
@@ -6,18 +6,18 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can install the AWS Load Balancer (ALB) Operator on a Security Token Service (STS) cluster.
+You can install the AWS Load Balancer Operator on a Security Token Service (STS) cluster.
 
-The ALB Operator relies on a `CredentialsRequest` to bootstrap the Operator and for the `AWSLoadBalancerController` instance. The ALB Operator waits until the required secrets are created and available.
+The AWS Load Balancer Operator relies on the `CredentialsRequest` object to bootstrap the Operator and the AWS Load Balancer Controller. The AWS Load Balancer Operator waits until the required secrets are created and available.
 
 [id="creating-iam-role-albo-operator_{context}"]
 == Creating an IAM role for the AWS Load Balancer Operator
 
-An additional IAM role is required to successfully install the ALB Operator on a Security Token Service (STS) cluster. The IAM role is required to interact with subnets and Virtual Private Clouds (VPCs). The Operator generates a `CredentialsRequest` with this role to bootstrap itself.
+An additional IAM role is required to successfully install the AWS Load Balancer Operator on a Security Token Service (STS) cluster. The IAM role is required to interact with subnets and Virtual Private Clouds (VPCs). The AWS Load Balancer Operator generates the `CredentialsRequest` object with the IAM role to bootstrap itself.
 
-There are two options for creating the IAM role:
+You can create the IAM role by using the following options:
 
-* Using `ccoctl` and a predefined `CredentialsRequest`.
+* Using the `ccoctl` binary and a predefined `CredentialsRequest` object.
 * Using the AWS CLI and predefined AWS manifests.
 
 Use the AWS CLI if your environment does not support the `ccoctl` command.
```

Comment:
maybe we could add link to ccoctl utility https://docs.openshift.com/container-platform/4.13/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts

Because in that page we could see the note that only Linux supports ccoctl.  
```
The ccoctl utility is a Linux binary that must run in a Linux environment.
```

Reply:
Done